pkg/constants/constants.go

```diff
@@ -23,7 +23,9 @@ var (
 
 // Controller Constants
 var (
-	ControllerLabelName = KFServingName + "-controller-manager"
+	ControllerLabelName        = KFServingName + "-controller-manager"
+	DefaultTimeout       int64 = 300
+	DefaultScalingTarget       = "1"
```

Comment:
Is this ScaleTarget or concurrency?

Reply:
https://github.com/knative/serving/blob/18d550c9051d337cb52e84137130c16e2e5da6cc/pkg/apis/serving/v1alpha1/revision_types.go#L194

Reply:
`containerConcurrency` field is for max concurrency used by queue proxy, default value is 0 which means unlimited. When `containerCurrency` is set to 0 it then looks into `scaleTarget` annotation for scaling, default `scaleTarget` is 100 I believe. The other choice is to set `containerCurrency` to 1 which is equivalent to set `containerCurrency=0` and `scaleTarget=1`, I think the later approach is more flexible since there can be use cases later user can tune the `scaleTarget` and we do not have to be too restrictive.

Reply:
This makes complete sense to me. We're *able* to send more than one request per replica, but the system will *attempt* to produce replicas to enforce an amortized one request per replica. Wow that's elegant.